web/src/services/Span.service.ts

```diff
@@ -92,7 +92,14 @@ export type TLintBySpanContent = {
   pluginName: string;
   passed: boolean;
   spanId: string;
-  errors: string[];
+  errors: {
```

Comment:
Do we have explicitly add the type here? or can we derive it from one of the auto-generated types?